### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dezzyboy**
a bit more details on getting started in development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
 interface that allows users to interact with the data on the BitClout blockchain.
+## Getting started
+Bitclout frontend-ui depends on node Node.js, currently tested on v14.16.1.
+
+On MacOs  and Unbuntu, the most reliable way to install Node is to use the [Node Version Manager](https://github.com/creationix/nvm) which allows you to have multiple node versions installed at once just incase you have other applications that depends on different versions.
+
+To install NVM for OSX/Linux, add the following command in your terminal:
+
+```
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.38.0/install.sh | bash
```
Comment (mywebpower) [inline README.md:12]: This seems like a really terrible idea blindly running shell scripts from unofficial urls.  nvm repo is here: https://github.com/nvm-sh/nvm

Comment (dezzyboy) [reply README.md:12]: sorry honest mistake will correct it asap with
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash

Comment (dezzyboy) [reply README.md]: added the official url

**Review by HPaulson — CHANGES_REQUESTED** [review]
Please rebase this to an updated main branch to remove the unwanted changes!

**Pull request closed without merging**